Update usage section of each component topic - master (#1584)
IgniteUI/igniteui-docfx: en/components/circular_progress.md

<div class="divider--half"></div>

### Usage

> Circular Progress Indicator can be used to show a user that he is in a process.

Reviewer: ```suggestion
Circular Progress Indicator can be used to show a user that they are in a process.
```
